app/controllers/rewards_controller.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
 class RewardsController < ApplicationController
+  REWARDS_PER_PAGE = 10
+
   def index
-    render :index, locals: { rewards: Reward.all }
+    @page = params.fetch(:page, 0).to_i
+    @number_of_page = (Reward.count / REWARDS_PER_PAGE).ceil
```

Comment:
I suggest renaming it to:
```suggestion
    @number_of_pages = (Reward.count / REWARDS_PER_PAGE).ceil
```

Reply:
I corrected the nomenclature